src/main/java/org/ow2/proactive/addons/ldap_query/LDAPClient.java

```diff
@@ -90,6 +90,7 @@ public LDAPClient(String ldapUrl, String ldapUsername, String ldapPassword, Stri
 
     public LDAPClient(Map<String, Serializable> actualTaskVariables, Map<String, Serializable> credentials) {
         ImmutableList<String> taskVariablesList = ImmutableList.of(ARG_URL,
```

Comment:
This constructor contains too much code, can we refactor it a bit by extracting small private methods with readable names :)